packages/guides-restructured-text/composer.json

```
@@ -22,7 +22,7 @@
     "minimum-stability": "stable",
     "require": {
         "php": "^8.1",
-        "doctrine/lexer": "^2.1",
+        "doctrine/lexer": "^3.0",
```

```suggestion
        "doctrine/lexer": "^2.1 || ^3.0",
```

Btw, I think only supporting doctrine/lexer 3 is fine (as far as I can see in https://github.com/phpDocumentor/phpDocumentor , only the guides package is using Doctrine lexer).

---

You're right, I will change that.

---

@wouterj I dropped `doctrine/lexer` 2 support again.